notebooks/loader_testing.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "70b1ee9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"src\")  #You might have to modify this according to the directory structure\n",
+    "\n",
+    "from load_bert import load_all, load_config_and_tokenizer, load_model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "324ae41d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_directory = \"/users/abhinandganesh/Desktop/bert_finetuned_sst2\" #Modify this according to where your model is"
```

Comment:
We can leverage Python here for everything expect the final file name. It would look something like this:
```
import os
root_dir = os.getcwd()
model_dir = os.path.join(root_dir, "models")
specific_model = os.path.join(model_dir, "YOUR_MODEL_NAME")
```

Python handles all the paths for you then, which is nicer than worrying about Windows / MAC / Linux etc. See cka.ipynb on develop for an example of this.